doc/doxygen/options.dox.in

```diff
@@ -70,7 +70,8 @@ WARN_IF_DOC_ERROR      = YES
 
 INPUT                  =
 RECURSIVE              = YES
-EXCLUDE_PATTERNS       = *.templates.h
+EXCLUDE_PATTERNS       = *.templates.h \
+                         *.txt
```

Comment:
I assume you want to specifically exclude the `CMakeLists.txt` files? If so, maybe just list that name, rather than using a wildcard.

Reply:
Seems to work.